lib/ssl/doc/src/ssl.xml

```diff
@@ -1610,12 +1624,14 @@ fun(srp, Username :: binary(), UserState :: term()) ->
     
     <func>
       <name name="clear_pem_cache" arity="0" since="OTP 17.5"/>
-      <fsummary> Clears the pem cache</fsummary>
+      <fsummary>Clears the pem cache</fsummary>
       <desc><p>PEM files, used by ssl API-functions, are cached. The
       cache is regularly checked to see if any cache entries should be
       invalidated, however this function provides a way to
       unconditionally clear the whole cache.
       </p>
+      <note><p>When PEM cache is enabled, calling this function ensures that current file content specified
+      with <c>cacertfile</c> option is used for following connection.</p></note>
       </desc>
```

Comment:
Sorry, it still sounds somewhat awkward (to me) :sweat_smile:

I don't think it should be separated out into a note (which is like "Oh, by the way, $this is what will also happen"), either. It describes what the function does, from a slightly different point of view, not a (possibly unexpected) side effect.

Consider this suggestion?
```suggestion
      <desc>
        <p>PEM files, used by ssl API-functions, are cached for
          performance reasons. The cache is automatically checked at
          regular intervals to see if any cache entries should be invalidated.</p>
        <p>This function provides a way to unconditionally clear the entire
          cache, thereby forcing a reload of previously cached PEM files.</p>
      </desc>
```